src/config/partners/showtime.ts

```diff
@@ -114,7 +114,7 @@ Wrap It Up with Incredible Raffles: Stick around to the end, and you could score
     {
       image: 'https://assets.onchainsummer.xyz/showtime-drop-7.mp4',
       creator: '0x6AcDf8958f7852720733A375e6accFa8Ec7C42d0',
-      name: `Performance Access Pass`,
+      name: `Talia Lahoud: Performance Access Pass`,
```

Comment:
I think "Access" is supposed to be removed based on the slack feedback?

Reply:
Thought that was an omission on our side. will check